pp_sql.gemspec

```diff
@@ -25,5 +25,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest-focus'
   s.add_development_dependency 'minitest-reporters'
   s.add_development_dependency 'rails', '>= 4.2'
-  s.add_development_dependency 'sqlite3', '~> 1.3.6'
+  s.add_development_dependency 'sqlite3'
```

Comment:
Pinning this gem to 1.3.6 was stopping me from running all of the tests. I think it's a version conflict with the Rails sqlite adapter?